steps/copywindowsefifilestowinpestagingonmacosorlinuxworker/script.txt

```diff
@@ -4,7 +4,7 @@ then
     cd "{automationWorkerLinuxBaseDirectory}/build-winpe-iso"
     
     # Copy EFI Files from Windows ISO
-    cp -rv "winimg/efi" "winpe_staging"
+    cp -rv "$(find winimg -type d -name efi)" "winpe_staging"

```

Comment:
This will cause problems as I've seen many efi directories in operating system ISOs

Why isn't this `winimg/*/efi` on both macOS and Linux?